src/gcinfo/CMakeLists.txt

```diff
@@ -5,13 +5,18 @@ set( GCINFO_SOURCES
   gcinfoencoder.cpp
 )
 
+
 if(CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
this should be CLR_CMAKE_TARGET_ARCH_I386 and then (not defined CLR_CROSS_COMPONENTS_BUILD) condition should go away